editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4957,6 +4987,7 @@ Node3DEditorViewport::Node3DEditorViewport(Node3DEditor *p_spatial_editor, int p
 	view_menu->get_popup()->add_shortcut(ED_GET_SHORTCUT("spatial_editor/right_view"), VIEW_RIGHT);
 	view_menu->get_popup()->add_shortcut(ED_GET_SHORTCUT("spatial_editor/front_view"), VIEW_FRONT);
 	view_menu->get_popup()->add_shortcut(ED_GET_SHORTCUT("spatial_editor/rear_view"), VIEW_REAR);
+	view_menu->get_popup()->add_check_item(TTR("View is +Z Forward"), VIEW_IS_Z_FORWARD);
```

Comment:
I'd rename the option to make it sound more natural:

```suggestion
	view_menu->get_popup()->add_check_item(TTR("+Z Is Forward"), VIEW_IS_Z_FORWARD);
```

Also, is this metadata stored for the current scene? If so, I'd add a tooltip to mention that the override only affects the current scene, not other scenes in the project.